tests/test_resolutions.py

```diff
@@ -187,12 +187,12 @@ def test_the_test(self) -> None:
         """Matches lists containing the item"""
         cti = ContainsTheItem(1).resolve()
 
-        assert cti.matches(range(0, 10))
+        assert cti.matches(range(10))
         assert not cti.matches({0, 3, 5})
 
     @pytest.mark.parametrize(
         ("arg", "expected"),
-        ((1, "A sequence containing <1>."), ("1", "A sequence containing '1'.")),
+        [(1, "A sequence containing <1>."), ("1", "A sequence containing '1'.")],
```

Comment:
I'm not sure I agree with their assessment of `parametrize` since the decorator is annotated to `Sequence`.

Reply:
Yeah, this tool enforces some `pytest` conventions settled on by the community. I agree with them so far, at least.